src/EA.Weee.Web/Areas/Aatf/Views/ManageEvidenceNotes/Overview/EditDraftReturnedNotesOverview.cshtml

```diff
@@ -96,11 +96,16 @@
                                     </td>
                                 </tr>
                             }
-
                         </tbody>
                     </table>
-
                     <br />
+                    @{
+                        Html.RenderPartial("~/Views/Shared/_EvidenceNotesPagerPartial.cshtml", Model.EvidenceNotesDataList, new ViewDataDictionary()
+                         { { "pagerRouteValues", new { tab = Model.ActiveOverviewDisplayOption.ToDisplayString(),
+                         manageEvidenceNoteViewModel_SelectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear,
+                         manageEvidenceNoteViewModel_RecipientWasteStatusFilterViewModel_NoteStatusValue =
```

Comment:
@GuidoWorkDev I dont think we need this route value setting as I dont think its a filter on this tab...

manageEvidenceNoteViewModel_RecipientWasteStatusFilterViewModel_NoteStatusValue